tests/test_incremental.py

```diff
@@ -6,9 +6,9 @@
 from dask.array.utils import assert_eq
 from sklearn.base import clone
 from sklearn.linear_model import SGDClassifier
+import numpy.linalg as LA
```

Comment:
The `LA` pronoun isn't very common.  I'd prefer the fully qualified name if possible.

```python
import numpy as np


np.linalg.svd(...)
```